packages/env/test/babel.config.js

```diff
@@ -0,0 +1,9 @@
+const path = require('path');
+
+module.exports = {
+  presets: [
+    require.resolve('@theforeman/builder/babel', {
```

Comment:
Are you sure `require.resolve('@theforeman/builder/babel')` won't be enough? (now you have it as a dep instead peerDep so it should fix it?)